OpenSim/Simulation/Model/Model.cpp

```diff
@@ -806,6 +807,10 @@ void Model::extendAddToSystem(SimTK::MultibodySystem& system) const
 {
     Model *mutableThis = const_cast<Model *>(this);
 
+    //Analyses are not Components so add them after legit 
+    //Components have been wired-up correctly.
+    mutableThis->updAnalysisSet().setModel(*mutableThis);
+
```

Comment:
Analysis is not a Component. `Analysis::setModel()` alerady assumes (e.g. JointReaction) that the Model is completely connected, which is not the case at the end of `Model::extendConnectToModel()`, since subcomponents haven't had a chance to do their thing.